crates/sncast/tests/helpers/constants.rs

```diff
@@ -32,3 +32,6 @@ pub const MAP_CONTRACT_DECLARE_TX_HASH_SEPOLIA: &str =
 
 pub const CONSTRUCTOR_WITH_PARAMS_CONTRACT_CLASS_HASH_SEPOLIA: &str =
     "0x59426c817fb8103edebdbf1712fa084c6744b2829db9c62d1ea4dce14ee6ded";
+
+pub const ARGENT_ACCOUNT_CLASS_HASH: &str =
+    "0x029927c8af6bccf3f6fda035981e765a7bdbf18a2dc0d630494f8758aa908e2b";
```

Comment:
where was it taken from?

Reply:
I just checked the class hash of the account that I deployed through Argent X. I couldn't find a specific class hash either in the Argent documentation or in their public repository. I tried to get some information from their Discord server but received no response. Notably, `starkli` also [utilizes this class hash](https://github.com/xJonathanLEI/starkli/blob/master/src/account.rs#L60), which is labeled as the `ArgentX` type on [Voyager](https://sepolia.voyager.online/class/0x029927c8af6bccf3f6fda035981e765a7bdbf18a2dc0d630494f8758aa908e2b).

Reply:
fine, thanks. I assume it is the same for all networks, right?